.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "window.title": "${activeEditorShort}${separator}${rootName}${separator}${profileName}${separator}[${activeRepositoryBranchName}]",
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  // For `sysoev.vscode-open-in-github` extension
+  "openInGitHub.defaultBranch": "unstable",
```

Comment:
As noted in previous comment, not a fan of putting this in the middle of `editor.` settings but I guess it's fine

Reply:
LOL... its funny that this PR had more discussion than a lot